docs/markdown/Reference-manual.md

```diff
@@ -1140,6 +1140,10 @@ Installs the entire given subdirectory and its contents from the
 source tree to the location specified by the keyword argument
 `install_dir`.
 
+If the subdirectory does not exist in the source tree, an empty directory is
+created in the specified location. A newly created subdirectory may only be
+created in the keyword argument `install_dir`.
+
```

Comment:
We should probably add a note that this is guaranteed to work after 0.57.0, and may work on older versions, just in case it turns out that didn't work in some ancient version of meson…

Reply:
I ran a simple script checking out every meson version and it works all the way back to including 0.45.0. Bisection then showed 8fe816101467e66792251b4f57e0ddddb537764a as the first commit supporting it.

I've added a `*(since 0.45.0)*` tag to the documentation now. I don't think we need to worry about stable updates to 2y old meson versions breaking this behaviour :)